css/style.css

```
@@ -1,4 +1,149 @@
-* {
-    margin: 0;
-    padding: 0;
-}
\ No newline at end of file
+
+
+ /*             CONTENT               */
+.content{
+    width: 100%;
+}
+
+
+input{
+    font-family: 'Roboto', sans-serif;
+    color: #333;
+    font-size: 1.2rem;
+    margin: 0 auto;
+    padding: 1.5rem 2rem;
+    border-radius: 0.2rem;
+    background-color: rgb(255, 255, 255);
+    border: none;
+    width: 50%;
+    display: block;
+    border-bottom: 0.3rem solid transparent;
+    margin-top:70px;
+}
+
+.text-for-typing{
+    margin: 0 auto;
+    width:500px;
+    font-size: 24px;
+    text-align: center;
+    margin-top:70px;
+    font-family:'Franklin Gothic Medium', 'Arial Narrow', Arial, sans-serif;
```

Please keep the spacings consistent

```suggestion
    width: 500px;
    font-size: 24px;
    text-align: center;
    margin-top: 70px;
    font-family:'Franklin Gothic Medium', 'Arial Narrow', Arial, sans-serif;
```